packages/standalone/standalone-cliplugins/README.md

```diff
@@ -0,0 +1,11 @@
+# standalone-cliplugins Package
+
+This standalone-cliplugins package is currently part of management package plugins to streamline the package release process. This package is not (and should not be) installed on the management cluster. Instead, it is used by the Tanzu CLI directly as an OCI bundle.
```

Comment:
this wording can be changed now that we have this package in a new repository

Reply:
updated